addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/state/SystemControlState.java

```diff
@@ -16,9 +16,11 @@
  */
 public class SystemControlState implements Invalidateable {
     public boolean power = false;
+    public boolean partyMode = false;
```

Comment:
The default value is already false so there's no need to initialize these values again to false.